dist/partials/query.parameter.html

```diff
@@ -54,6 +73,11 @@
 				<li>{{metric.key1.key2}}</li>
 				<li>{{resource.key1.key2}}</li>
 			</ul>
+			Examples:
+			<ul ng-non-bindable>
+				<li>Server {{metric.labels.instance_name}}</li>
```

Comment:
Should this be called "server"? Does that concept map naturally onto `instance_name`?

Reply:
This is just an example, "metric.labels.instance_name" can mean VM name which can be interpreted as server name. I'm open to suggestion for clearer example though.